src/pages/workspaces/workspace/Data.js

```diff
@@ -601,6 +612,13 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
         workspace,
         selectedEntities: _.keys(selectedEntities), selectedDataType: entityKey, runningSubmissionsCount
       }),
+      viewData && h(Modal, {
+        title: 'Contents',
+        showButtons: false,
+        showX: true,
+        contentStyle: { display: 'flex', flexDirection: 'column', maxHeight: '90%', overflowY: 'auto' },
```

Comment:
FYI: Another way to do this is to use `vh` units, which correspond to the viewport height. E.g. we could have a scroll container with something like `80vh`, which could be done without changing the base modal.